### PR TITLE
Mention commiting only bower.json

### PR DESCRIPTION
**Pull request by wouterj**
Replaces https://github.com/symfony/symfony-docs/pull/5778

| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

The section is called "Should I Git Ignore or Commit Bower Assets?", but we cover only the commit way and not the git ignore way, giving users a bit biased information. We should at least mention that people can also commit bower.json.

Thanks to @krike for providing the original patch!

### CODE REVIEW

File: cookbook/frontend/bower.rst [hunk 1]
```diff
@@ -135,6 +135,12 @@ For more details, read the article `Checking in front-end dependencies`_.
 But, it's very possible that Bower will add a lock feature in the future
 (e.g. `bower/bower#1748`_).
 
+If you don't care too much about having *exact* the same versions, you can only
+commit the ``bower.json`` file. Running ``bower install`` will give you the
+latest versions within the specified version range of each package in
+``boewr.json``. Using strict version constraints (e.g. ``1.10.*``) is often
```
Comment (hansstevens) [inline cookbook/frontend/bower.rst]: small typo there

**Comment by krike** [comment]
Excellent :D.

You do have a typo on line 141 `boewr.json` :)

File: cookbook/frontend/bower.rst (hunk 1 above)
Comment (WouterJ) [inline cookbook/frontend/bower.rst]: Fixed, thanks!

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @krike.

**Pull request merged**